### DISCUSSION TITLE
Create Discord server (or similar)

### DISCUSSION THREAD

**Discussion by Bukama** [support-and-care/maven-support-and-care#29] [Ideas for Funding]
As of now there is the ASF slack, for which users need invitations (I'll get mine soon thanks to @bmarwell ), but I think a more public possibility for a "direct" chat to ask quick questions should be thought about.

For example I'm on the PrimeTek Discord server sitting in the Primefaces channel there and there are fast to answer / or "please create an issue with a reproducer that seems like a bug" questions / answers popping up my users (answers obviously by the maintainers) without being noisy. The chat also helps to get in contact with other peoples who use Maven.

Some persons might argue that the maven user list exists for such things, but I thing a user lists and a chat get other people involved.

**Comment by rmannibucau** [comment]
Think it would be the promise of a degraded communication.
Slack already has some negative impact but has the advantage to let anyone speaks to maven guys in realtime but far from all core committers are there and at asf only counts.
So long story short, to stay good we must keep a single realtime solution which is slack today.
It can be discord or gitter if slack is dropped but it is unlikely to happen soon IMHO.

**Reply by hendrikebbers** [reply]
What about Github Discussions? Looks like it is working quite good for the current use case. Could be activated for https://github.com/apache/maven

**Reply by rmannibucau** [reply]
Guess the same applies. Ok if we drop lists which is not really an option at asf.
Let me ask maybe which issue this thread is about since we already have async, realtime 1nd bugtracker comm?

**Reply by Bukama** [reply]
my thought about the Discord is about the (ofc who's online) sync communication/chat esp for quick answers, which (at least I would never put on Github discussions or similar)

**Reply by hendrikebbers** [reply]
But you would on slack, right? But with slack you can not join without an invite, correct?

**Reply by rmannibucau** [reply]
Yep and requesting an invite is as trivial as a mail so dont think it is a big blocker which avoids to spread the communication and community.

**Comment by Bukama** [comment]
> But you would on slack, right? But with slack you can not join without an invite, correct?

Yes access to ASF slack is either by having an `@apache` mail (so be ASF member) or with invite from someone at ASF.
